pyrefly.toml

```diff
@@ -142,3 +142,11 @@ errors.deprecated = false # re-enable after we've fix import formatting
 permissive-ignores = true
 replace-imports-with-any = ["!sympy.printing.*", "sympy.*", "onnxscript.onnx_opset.*"]
 search-path = ["tools/experimental"]
+
+[[sub-config]]
+matches = "torch/_dynamo/**"
+# All errors enabled by (default)
+[sub-config.errors]
+bad-argument-count = false
+implicit-import = false
```

Comment:
implicit-import is caused by the `from x import .`.  We will try turning this on in a followup